yogstation/code/modules/scripting/Parser/Expressions.dm

```diff
@@ -166,12 +166,12 @@
 	- <ParseParamExpression()>
 */
 		ParseExpression(list/end=list(/token/end), list/ErrChars=list("{", "}"), check_functions = 0, check_assignments = 1)
-			var/stack
-				opr=new
-				val=new
+			var/stack/opr = new
+			var/stack/val = new
+
```

Comment:
Don't edit like 3 mutually exclusive modules of the game in one PR.

Make your NTSL changes and whatever else separate from this Xenobio stuff.

Reply:
it broke my linter so i had to fix that,do i really need to make another PR for each thing i fix thats not related to this PR but literally doesnt impact a thing in the game

Reply:
Yes